src/app/haku/[oid]/hakukohde/[hakukohde]/sijoittelun-tulokset/page.tsx

```diff
@@ -1,19 +1,95 @@
 'use client';
 
-import { useHakukohde } from '@/app/hooks/useHakukohde';
 import { TabContainer } from '../components/tab-container';
+import { useTranslations } from '@/app/hooks/useTranslations';
+import { QuerySuspenseBoundary } from '@/app/components/query-suspense-boundary';
+import { Box } from '@mui/material';
+import { useSuspenseQuery } from '@tanstack/react-query';
+import { getLatestSijoitteluAjonTuloksetWithValintaEsitys } from '@/app/lib/valinta-tulos-service';
+import { isEmpty } from '@/app/lib/common';
+import { PageSizeSelector } from '@/app/components/table/page-size-selector';
+import { NoResults } from '@/app/components/no-results';
+import { useSijoittelunTulosSearchParams } from './hooks/useSijoittelunTuloksetSearch';
+import { SijoittelunTulosContent } from './components/sijoittelun-tulos-content';
+import { SijoittelunTulosControls } from './components/sijoittelun-tulos-controls';
+import { useHaku } from '@/app/hooks/useHaku';
+import { Haku } from '@/app/lib/types/kouta-types';
+import { ClientSpinner } from '@/app/components/client-spinner';
+
+type SijoitteluContentParams = {
+  haku: Haku;
+  hakukohdeOid: string;
+};
+
+const SijoitteluContent = ({ haku, hakukohdeOid }: SijoitteluContentParams) => {
+  const { t } = useTranslations();
+
+  const { pageSize, setPageSize } = useSijoittelunTulosSearchParams();
+
+  const { data: tulokset } = useSuspenseQuery({
+    queryKey: [
+      'getLatestSijoitteluAjonTuloksetWithValintaEsitys',
+      haku.oid,
+      hakukohdeOid,
+    ],
+    queryFn: () =>
+      getLatestSijoitteluAjonTuloksetWithValintaEsitys(haku.oid, hakukohdeOid),
+  });
+
+  return isEmpty(tulokset) ? (
+    <NoResults text={t('hakijaryhmat.ei-tuloksia')} />
+  ) : (
+    <Box
```

Comment:
Näitä sisäkkäisiä bokseja tässä leiskassa riittää...en tiedä tulisiko tästä box-hässäkästä luettavampi jos irrottais ainakin jonkun wrapperiboxin komponentiksi vaikka sitten saman tiedoston sisälle.

Reply:
Tähän oli jäänyt yksi ylimääräinen, poistin sen